added a website in buttons category

Blurri10 opened the pull request.

juxtopposed commented:
Hey! It's good, but since there's no gif, you don't need to include another image for gif. One would be enough.

Blurri10 commented:
actually i wanted to add that gif file, but everyone else had added webp files for gifs. so i converted my gif file into webp and added it. i searched more about webp and it did support animated images, but mine doesn't work 😭. idk how everyone posted gif as webp. (sorry for the inconvenience)

Blurri10 commented:
okay i fixed it. that webp works properly now. should i close this request and submit another pull request with that gif?
